articles/manual/statements.md

```diff
@@ -0,0 +1,928 @@
+- **Type Parameters**: An empty section which contains one additional
+  subsection for each generic type parameter.
+- **Example**: A short example of the operation, function or type in use.
+- **Remarks**: Miscellaneous prose describing some aspect of the operation,
+  function, or type.
+- **See Also**: A list of fully qualified names indicating related functions,
+  operations, or user-defined types.
+- **References**: A list of references and citations for the item being
+  documented.
+
+## Namespaces
+
+Every Q# operation, function, and user-defined type is
+defined within a namespace.
+Q# follows the same rules for naming as other .NET languages.
+However, Q# does not support relative references to namespaces.
+That is, if the namespace `a.b` has been opened, a reference to an operation
+names `c.d` will not be resolved to an operation with full name `a.b.c.d`.
+
+Within a namespace block, the `open` directive may be used to 
+import all types and callables declared in a certain namespace and refer to them by their unqualified name. Optionally, a short name for the opened namespace may be defined such that all elements from that namespace can (and need) to be qualified by the defined short name. 
+The `open` directive applies to the entire namespace block within a file.
+
+A type or callable defined in another namespace that is not
+opened in the current namespace must be referenced by its fully-qualified name.
+For example, an operation named `Op` in the `X.Y` namespace must be
+referenced by its fully-qualified name `X.Y.Op`, unless the `X.Y`
+namespace has been opened earlier in the current block. 
+As mentioned above, even if the `X` namespace has been opened, it is not possible to reference the operation as `Y.Op`.
+If a short name `Z` for `X.Y` has been defined in that namespace and file, then `Op` needs to be referred to as `Z.Op`. 
+
+```qsharp
+namespace NS {
+    open Microsoft.Quantum.Intrinsic; // opens the namespace
+    open Microsoft.Quantum.Math as Math; // defines a short name for the namespace
+}
+```
+
+It is usually better to include a namespace by using an `open` directive.
+Using a fully-qualified name is required if two namespaces define constructs
+with the same name, and the current source uses constructs from both.
+
+## Formatting
+
+We are working to ensure that common punctuation marks are used consistently in all situations.
+We expect that this will make Q# easier to learn and to read because these marks always mean the same thing, and therefore the same concept is always represented the same way.
+
+Specifically:
+
+- The semi-colon, `;`, is used to end a statement or single-line directive.
+	It is not used for any other purpose.
+- The comma, `,`, is used to separate elements of a sequence. It is used for tuple literals, array literals, argument lists, tuple definitions, and type lists. 
+	**In a change from earlier versions, `;` is no longer supported as an array literal separator.**
```

Comment:
This change was quite a while ago, do we want to keep that note? (+@bettinaheim for input)